src/app/tab/time/time.component.html

```diff
@@ -1,111 +1,289 @@
+        xml:space="preserve"
+        [style.fill]="settings.config.design.foreground"
+      >
+        <circle cx="250" cy="11.4" r="11.4" />
+        <ellipse
+          transform="matrix(0.5 -0.866 0.866 0.5 147.13 341.5063)"
+          cx="369.3"
+          cy="43.3"
+          rx="11.4"
+          ry="11.4"
+        />
+        <ellipse
+          transform="matrix(0.866 -0.5 0.5 0.866 -4.1594 245.8406)"
+          cx="456.7"
+          cy="130.7"
+          rx="11.4"
+          ry="11.4"
+        />
+        <circle cx="488.6" cy="250" r="11.4" />
+        <ellipse
+          transform="matrix(0.5 -0.866 0.866 0.5 -91.5063 580.1427)"
+          cx="456.7"
+          cy="369.3"
+          rx="11.4"
+          ry="11.4"
+        />
+        <ellipse
+          transform="matrix(0.866 -0.5 0.5 0.866 -178.8533 245.8406)"
+          cx="369.3"
+          cy="456.7"
+          rx="11.4"
+          ry="11.4"
+        />
+        <circle cx="250" cy="488.6" r="11.4" />
+        <ellipse
+          transform="matrix(0.5 -0.866 0.866 0.5 -330.1427 341.5063)"
+          cx="130.7"
+          cy="456.7"
+          rx="11.4"
+          ry="11.4"
+        />
+        <ellipse
+          transform="matrix(0.866 -0.5 0.5 0.866 -178.8533 71.1467)"
+          cx="43.3"
+          cy="369.3"
+          rx="11.4"
+          ry="11.4"
+        />
+        <circle cx="11.4" cy="250" r="11.4" />
+        <ellipse
+          transform="matrix(0.5 -0.866 0.866 0.5 -91.5063 102.87)"
+          cx="43.3"
+          cy="130.7"
+          rx="11.4"
+          ry="11.4"
+        />
+        <ellipse
+          transform="matrix(0.866 -0.5 0.5 0.866 -4.1594 71.1467)"
+          cx="130.7"
+          cy="43.3"
+          rx="11.4"
+          ry="11.4"
+        />
       </svg>
-      <div class="analogCenter"
-        [style.background]="settings.config.design.foreground">
-      </div>
-      <div class="analogHour"
-        [ngStyle]="{'transform': getAnalogHour(clock.timezone)}"
-        [style.background]="settings.config.design.foreground">
+      <div
+        class="analogCenter"
+        [style.background]="settings.config.design.foreground"
+      ></div>
+      <div
+        class="analogHour"
+        [ngStyle]="{ transform: getAnalogHour(clock.timezone) }"
+        [style.background]="settings.config.design.foreground"
+      ></div>
+      <div
+        class="analogMinute"
+        [ngStyle]="{ transform: getAnalogMinute(clock.timezone) }"
+        [style.background]="settings.config.design.foreground"
+      ></div>
+      <div
+        class="analogSecond"
+        *ngIf="clock.seconds.enabled"
+        [ngClass]="{ dim: clock.seconds.dim }"
+        [ngStyle]="{ transform: getAnalogSecond(clock.timezone) }"
+        [style.background]="settings.config.design.foreground"
+      ></div>
+    </div>
+  </div>
+  <div
+    class="binary"
+    *ngIf="clock.binary.enabled"
+    [style.fontSize]="shared.getFontSize(clock.delimiter.scaling, 1.2)"
```

Comment:
It is not obvious from the options that the size of the binary clock is based on delimiter scaling, this should be based on the clock's overall size

Reply:
Changed the binary cock to use the clock.size properties to set the size of the dab's of the binary clock

https://user-images.githubusercontent.com/2345436/121958100-92021b00-cd63-11eb-873f-d868fa3d7ad8.mp4